docs/src/equations.md

```diff
@@ -27,7 +27,10 @@ We make use of the following operators
 
 * ``I^c`` is the [face-to-center reconstruction operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.InterpolateF2C)
 * ``I^f`` is the [center-to-face reconstruction operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.InterpolateC2F)
-  - Currently this is just the arithmetic mean, but we will need to use a weighted version with stretched vertical grids.
+  - Currently this is just the arithmetic mean, for uniform grids. 
+* ``WI^f`` is the [center-to-face weighted reconstruction operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.WeightedInterpolateC2F)
+  - This defines the generalised weighted interpolation operator, for use with stretched grids. 
+  - ``J`` is the value of the Jacobian for use in the weighted interpolation operator
 * ``U^f`` is the [3rd-order center-to-face upwind product operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.Upwind3rdOrderBiasedProductC2F)
```

Comment:
Also add function signature. Additionally, this is confusing because it states 3rd-order upwind in the first line, then 1st-order upwind below. Be clear about what this is and how the order is determined.

Reply:
More comments on the sections below:

Under "Differentiation operators" (perhaps better: Differential operators)
* Vertical divergence, fluxes set to zero and top and bottom: Is that still the case? I thought we have diffusive fluxes at the bottom now. How are they handled?
* C^f_v is the ... vertical curl: This is misleading I think. It seems to me this is a *horizontal* curl, i.e., a curl component of the vertical velocity (shear) that is entirely horizontal. (It is a curl of a vertical velocity, but a horizontal, not vertical, curl, isn't it?)

Under Auxiliary and derived quantities:
* Add s^{-1} after 7.29212 \times 10^{-5} for planetary rotation rate.
* Kinetic energy needs to be updated (or at least flagged as problematic)